.github/workflows/ci-prerelease.yml

```diff
@@ -70,3 +70,22 @@ jobs:
       AWS_S3_BUCKET_NAME: "nr-downloads-ohai-testing"
       AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock-testing"
       ASSETS_TYPE: "all"
+
+  packaging-tests:
+    runs-on: ubuntu-20.04
+    needs: [publishing-to-s3]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set and validate distribution name and go-version
+        run: .github/workflows/scripts/set_version.sh
+
+      - uses: newrelic/pkg-installation-testing-action@v1
+        with:
```

Comment:
Is there any reason not to run the `component_molecule_packaging.yml`?

Reply:
Yes, it seems that is not possible to run reuse local workflows in already started jobs. Note that  `component_molecule_packaging.yml` defines the instance that it will run, which it can be different that the actual running.
The issue in this case is that we need to check out in the actual repository to get the version details. 
Initially I use the component file, but this is the error that I got: https://github.com/newrelic/opentelemetry-collector-releases/actions/runs/4025196671/jobs/6918579337